warehouse/ingest-core/src/main/java/datawave/ingest/data/config/ingest/ContentBaseIngestHelper.java

```diff
@@ -31,6 +31,9 @@
     private final Set<String> contentIndexBlacklist = new HashSet<>();
     private final Set<String> contentReverseIndexBlacklist = new HashSet<>();
     
+    private final Set<String> indexListEtriesFields = new HashSet<>();
```

Comment:
I know you probably do not need it immediately, but do you thing a blacklist/whitelist mechanism would be useful as we do with the content indexing (e.g. tokenizing)?

Reply:
This is viewed as a select set of fields that would require to be treated as a list of values. I can't see a scenario where most of the fields of a "feed" are a list of values.